tests/test_run_inputs.py

```diff
@@ -22,6 +21,6 @@ def test_inputs():
         if cyclus_has_coin() or "GrowthRegion" not in src:
```

Comment:
(This comment applies to the `for` loop above that I can't comment on here.)

Can we replace this loop with a `pytest` fixture? It has the advantage of making each file operate as a different test.

Reply:
Implemented a fixture to loop through all the test files.  Pytest now treats each one as a different test (confirmed in pytest's output by the 16 passing dots instead of 1)